benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -141,10 +141,19 @@ void GraphicsBenchmarkApp::Setup()
         samplerCreateInfo.maxLod                  = FLT_MAX;
         PPX_CHECKED_CALL(GetDevice()->CreateSampler(&samplerCreateInfo, &mLinearSampler));
     }
+    // Descriptor Pool
+    {
+        grfx::DescriptorPoolCreateInfo createInfo = {};
```

Comment:
why do you need 100 for each?

Reply:
Reduced to 10

Reply:
sorry still a bit confused, why is it 10? according to the comments, should sampler be 4, sampledImage be 5 and uniformbuffer be 2?

Reply:
I thought it was acceptable to allocate a bit extra, also I wasn't sure about if # of frames in flight would change. But I see now that it's a setting so I've added that explicitly in `Config` and I've reduced the descriptor counts to 4/5/2.

Reply:
or would it be better to let each one multiply by GetNumFramesInFlight()?

Reply:
Sounds good, fixed